aspnetcore-webapi/GettingStartedASPNETMongoDB/GettingStartedASPNETMongoDB/Interfaces/ICourseService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using GettingStartedASPNETMongoDB.Models;
+
+namespace GettingStartedASPNETMongoDB.Interfaces
+{
+    public interface ICourseService
+    {
+        Task<Course> Create(Course course);
+
+        Task<Course> GetById(string id);
```

Comment:
If this can return null when no Course is found, we should mark the return type as nullable with that Task<Course?>